backend/curriculum_tracking/models.py

```diff
@@ -941,6 +947,28 @@ def get_activity_log_summary_data(self):
             "flavour_names": self.content_item.flavour_names,
         }
 
+    @property
+    def duration_in_current_column(self):
+        relevant_logs = sorted(
```

Comment:
Please remove this function.  If you find yourself putting the same code in 2 different PRs then something has gone wrong.  Please keep your git tidy.

I dont think we ever had duration information show up on the frontend here. Why is it being added into this PR?

Reply:
@sheenarbw, I was displaying duration information because of this point in the task spec:
![Screenshot 2024-04-07 124924](https://github.com/Umuzi-org/Tilde/assets/94048861/aa01a3fa-9626-472c-9b85-f1da0b0da6ca)